CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# Contributing To A Model Deployment Repository
```

Comment:
We also want users that simply want to modify the source and do a test-run to read this document even if they don't intend to contribute source code.

Reply:
Updated header in https://github.com/ACCESS-NRI/model-deployment-template/pull/9/commits/ba6dd8d3e353f6b3dd9c8efe9207f95af6f6ac51